### ISSUE TITLE
declarations inside constructor not renamed

### ISSUE THREAD

**Issue by dariajung** [angular/ts-minify#35]
```
Parameter properties
The keywords 'public' and 'private' also give you a shorthand for creating and initializing members of 
your class, by creating parameter properties. 
The properties let you can create and initialize a member in one step.
 Here's a further revision of the previous example.
 Notice how we drop 'theName' altogether and just use the shortened 'private name: string' parameter on the constructor to create and initialize the 'name' member.
```

**Comment by dariajung** [comment]
Properties can be declared and initialized inside the constructor. Currently unsupported, which breaks when the user of the minifier tool uses this syntax.

**Comment by dariajung** [comment]
Working through this on branch `parameter-properties`

**Comment by dariajung** [comment]
addressed with ad68fb64db720a5791f34eafce5bb532268f151b, but new related issue #39
